lib/lib_rover/include/rover_helpers/assert.hpp

```diff
@@ -52,13 +66,13 @@
         {                                 \
             LOG(FATAL, "%s", #condition); \
             Serial.flush();               \
-            abort();                      \
+            ABORT();                      \
         }                                 \
     }
 
 #define ASSERT0() \
     {             \
-        abort();  \
+        ABORT();  \
     }
 
```

Comment:
Maybe make ASSERT (0-1-2) define names more comlient to different utilities

Reply:
done